<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="san3149591" ignored="0" name="Test Cases for users" tests="4" failures="1" timestamp="2020-02-03T15:13:24 IST" time="19.789" errors="0">
  <testcase name="verifyToGetUserID" time="4.762" classname="com.freenow.testcases.TC01_Users"/>
  <testcase name="verifyToUserEmailAddress" time="0.666" classname="com.freenow.testcases.TC01_Users"/>
  <testcase name="createNewUser" time="7.922" classname="com.freenow.testcases.TC01_Users"/>
  <testcase name="getCreateNewUserDetails" time="5.633" classname="com.freenow.testcases.TC01_Users">
    <failure type="java.lang.AssertionError" message="response is not 200 expected [404] but found [200]">
      <![CDATA[java.lang.AssertionError: response is not 200 expected [404] but found [200]
at com.freenow.testcases.TC01_Users.getCreateNewUserDetails(TC01_Users.java:159)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at com.sun.proxy.$Proxy0.invoke(Unknown Source)
... Removed 39 stack frames]]>
    </failure>
  </testcase> <!-- getCreateNewUserDetails -->
</testsuite> <!-- Test Cases for users -->
